lib/srv/alpnproxy/common/protocols.go

```diff
@@ -63,7 +63,7 @@ var supportedProtocols = []Protocol{
 	ProtocolAuth,
 }
 
-func convProtocolsToString(protocols []Protocol) []string {
+func ConvProtocolsToString(protocols []Protocol) []string {
```

Comment:
Nit: Add godoc and also, maybe "Conv" is redundant in the name? Just `ProtocolsToString` works well too.